python/mxnet/optimizer.py

```diff
@@ -1091,14 +1091,20 @@ class AdaGrad(Optimizer):
     ----------
     eps: float, optional
         Small value to avoid division by 0.
+    initial_accumulator_value: float, default 0
+        The Adagrad state is initially set to this value.
 
     """
-    def __init__(self, eps=1e-7, **kwargs):
+    def __init__(self, eps=1e-7, initial_accumulator_value=0, **kwargs):
```

Comment:
not sure about the name.

Reply:
I think it comes from https://github.com/tensorflow/tensorflow/blob/r1.8/tensorflow/python/training/adagrad.py#L46